README.md

```diff
@@ -40,20 +69,33 @@ and [iOS Platform Guide](https://cordova.apache.org/docs/en/4.0.0/guide_platform
 ### Android Requirements
 
 ### Use Gradle
```

Comment:
The indenting on line 71, 75, etc. is wrong. These are all instructions for Android. So Line 71 should be "#### use Gradle" and 75 should be "#### Build the P2P library to local Maven"and the children of that should be 5 #s not 4.